### DISCUSSION TITLE
No longer able to unzoom

### DISCUSSION THREAD

**Discussion by MD59df** [toutjavascript/Fooocus-Log-Viewer#24] [General]
I tried to re-install the files over and over, but i'm stuck at max zoom, the buttons no longer works, any advice?

Ok, so i found out the problem, when you over unzoom, it loops baack to the maximum zoom state. So i was just stuck at a lot of unzooming

**Comment by toutjavascript** [comment]
Hello

Thanks for analyze. I will add a max unzoom value on button
